docs/how-tos/collect-sensor-data.md

```diff
@@ -15,20 +15,23 @@ You can use the data management service to capture sensor data from any or all o
 For example, you can configure data capture for several sensors across one or multiple machines to report the current memory usage or the ambient operating temperature.
 
 You can do all of this using the [Viam app](https://app.viam.com/) user interface.
-You will not need to write any code.
+You do not need to write any code to follow this guide.
 
 {{< alert title="In this page" color="tip" >}}
 
 1. [Gather data on any machine and syncing it to the cloud](#gather-and-sync-data).
 1. [View sensor data](#view-sensor-data).
+1. [Stop data capture](#stop-data-capture).
 
 {{< /alert >}}
 
 ## Prerequisites
 
+You don't need to buy or own any hardware to complete this tutorial.
```

Comment:
```suggestion
You don't need to buy or own any hardware to complete this tutorial.
You do need:
```
[opt] I think some sort of transition would be good here to go from what you don't need to what you do in fact need.